content/en/news/releases/1.6.x/announcing-1.6/_index.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Announcing Istio 1.6
+linktitle: 1.6
+subtitle: Major Update
+description: Istio 1.6 release announcement.
+publishdate: 2020-05-19
+release: 1.6.0
+skip_list: true
+aliases:
+    - /news/announcing-1.6.0
+    - /news/announcing-1.6
+---
+
+We are pleased to announce the release of Istio 1.6!
+
+{{< relnote >}}
+
+With this release, we are continuing the path we charted earlier this year in
+our [roadmap post](/blog/2020/tradewinds-2020/), sailing toward more
+simplicity, a better installation experience, and some other goodies as well.
+
+Here’s some of what’s coming to you in today's release:
+
+## Simplify, simplify, simplify
+
+Last release, we introduced **Istiod**, a component that reduced the number of
+components in an Istio installation by combining the functionality of several
+other components. In Istio 1.6, we have completed this transition and have fully
+moved functionality into Istiod, allowing us to remove Citadel, the sidecar
+injector, and Galley are no longer deployed.
+
+You'll also find that we are using the new `appProtocol` field in the Kubernetes
+[`EndpointPort`](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#endpoints-v1-core)
+API. What does that mean for you? It means that you won't need to use the name
+field in your service to denote the protocol -- a simpler user experience.
+
+## Better lifecycle
+
+We continue to make installing and upgrading Istio a better experience. Our
+command line tool `istioctl` gives better diagnostic information, has a simpler
+install command, and even gives status in color!
+
+Upgrading Istio has been improved as well, in several powerful ways. First, we
+now support canarying of the Istio control plane itself. That means you can
+install a new version of the control plane alongside the existing version and
+selectively have proxies use the new one.
+
+We also have an `istioctl upgrade` command that will perform an in-place
+upgrade in your clusters (still giving you the control over updating the proxies
+themselves).
+
+Check out the [documentation](/docs/setup/upgrade/) for all of the details on
+the new upgrade experience.
+
+## Observe this
+
+Many companies adopt Istio solely to get better observability of distributed
+applications, so we continue to invest there. You'll have to read the release
```

Comment:
should this link to the release notes?

Reply:
YES! Will fix.

Reply:
Fixed, and also there will be a link at the top autogenerated like the table you see on https://istio.io/news/releases/1.5.x/announcing-1.5/